lib/actions/updateDispatchDate.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const fetch = require('node-fetch');
+const requestOptions = require('../requestOptions.js');
+
+exports.process = function(msg, cfg) {
+  let self = this;
+  let orderId = msg.body.webhook.data.id;
+  let dispatchDate = msg.body.electio.dispatchDate;
+
+  function updateOrder(authenticationHeaders) {
+     let requestBody = {
+       data: {
+         id: orderId,
+         type: "orders",
+         attributes: {
+           "dispatch-date": dispatchDate
+         }
+       }
+     };
+
+     if (dispatchDate !== null) {
```

Comment:
I would have `if (dispatchDate)` here instead, which will be false if the dispatch date is `null`, `undefined`, an empty string, etc.